src/EPR.Calculator.API.UnitTests/CreateDefaultParameterSettingMapperTest.cs

```diff
@@ -0,0 +1,86 @@
+﻿using api.Dtos;
+using api.Mappers;
+using EPR.Calculator.API.Dtos;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace api.Tests.Controllers
+{
+    [TestClass]
+    public class CreateDefaultParameterSettingMapper : BaseControllerTest
+    {
+        private static string[] _uniqueReferences = {"BADEBT-P", "COMC-AL", "COMC-FC", "COMC-GL",
```

Comment:
@rekhadhankhar2018 - this can be moved into a separate static class, for example, we can create a Mocks folder and create a static class in it to return these values.

This is just a suggestion and does not need to be done in this ticket.

Reply:
implemented that John